src/command/history.rs

```diff
@@ -2,60 +2,51 @@ use std::env;
 use std::io::Write;
 use std::time::Duration;
 
+use clap::Subcommand;
 use eyre::Result;
-use structopt::StructOpt;
 use tabwriter::TabWriter;
 
 use atuin_client::database::Database;
 use atuin_client::history::History;
 use atuin_client::settings::Settings;
 use atuin_client::sync;
 
-#[derive(StructOpt)]
+#[derive(Subcommand)]
+#[clap(infer_subcommands = true)]
 pub enum Cmd {
-    #[structopt(
-        about="begins a new command in the history",
-        aliases=&["s", "st", "sta", "star"],
-    )]
+    /// Begins a new command in the history
     Start { command: Vec<String> },
 
-    #[structopt(
-        about="finishes a new command in the history (adds time, exit code)",
-        aliases=&["e", "en"],
-    )]
+    /// Finishes a new command in the history (adds time, exit code)
     End {
         id: String,
-        #[structopt(long, short)]
+        #[clap(long, short)]
         exit: i64,
     },
 
-    #[structopt(
-        about="list all items in history",
-        aliases=&["l", "li", "lis"],
-    )]
+    /// List all items in history
     List {
-        #[structopt(long, short)]
+        #[clap(long, short)]
         cwd: bool,
 
-        #[structopt(long, short)]
+        #[clap(long, short)]
         session: bool,
 
-        #[structopt(long, short)]
+        #[clap(long, short)]
         human: bool,
```

Comment:
Seems this is having some troubles. It's inferring the subcommand as `-h` but it's conflicting with the help option. There might be other cases like this too.

I guess an easy solution would be to extract this list variant into a new struct

Reply:
I think we have to make a decision here; do we want `atuin history last -h` to be `atuin history last --help` or `atuin history last --human`?

Reply:
I'd go for -h being help, it's more intuitive and in line with what people might expect

Reply:
Agreed